python/paddle/v2/fluid/tests/test_warpctc_op.py

```diff
@@ -191,10 +191,10 @@ def setUp(self):
     def test_check_output(self):
         self.check_output()
 
+    def test_check_grad(self):
+        self.outputs['WarpCTCGrad'] = self.gradient
+        self.check_grad(["Logits"], "Loss", max_relative_error=0.01)
```

Comment:
Can the gradient check pass when `max_relative_error` is lower?

Reply:
It will fail while `max_relative_error = 0.005`.

Reply:
Change `max_relative_error` from 0.01 to 0.007.